internal/tiltfile/docker.go

```diff
@@ -390,61 +376,6 @@ func parseValuesToStrings(value starlark.Value, param string) ([]string, error)
 	return ignores, nil
 
 }
-func (s *tiltfileState) fastBuild(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
```

Comment:
again: leave this method in but have it throw a deprecation error with a link to LiveUpdate docs?